packages/shared/src/libs/types/pagination-response.type.ts

```diff
@@ -0,0 +1,5 @@
+type PaginationResponse = {
+	totalItems: number;
+};
```

Comment:
Let's make it a generic 
```suggestion
type PaginationResponseDto = <T>{
    items: T[];
	totalItems: number;
};
```